src/ToDoHeader/ToDoHeader.css

```diff
@@ -0,0 +1,28 @@
+.ToDoHeader {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 5px;
+  background-color: var(--blue-color-4);
+  border-bottom: var(--spacing-0) solid var(--brown-color-2);
+  min-height: 100px;
+
+  --subtitle-size: var(--spacing-3);
+}
+
+.ToDoHeader-title {
+  margin: var(--spacing-3) 0;
```

Comment:
Looks like there might be padding and margin above the `title`. I would probably stick with only one and apply on the parent element.